ci/Dockerfile.std

```diff
@@ -1,36 +1,31 @@
 
-
-# Generated by: tyk-ci/wf-gen
-# Generated on: Friday 04 March 2022 11:40:32 AM UTC
-
-# Generation commands:
-# ./pr.zsh -repos tyk -base td-883-test -branch td-883-test -title Sync from latest releng templates
-# m4 -E -DxREPO=tyk
+# Generated by: gromit policy
+# Generated on: Mon Jul 11 07:20:30 UTC 2022
 
 FROM debian:bullseye-slim
 ARG TARGETARCH
 
 ENV DEBIAN_FRONTEND=noninteractive
 
+RUN apt-get update \
+    && apt-get dist-upgrade -y ca-certificates
 # Install curl and python3
-RUN apt update \
-    && apt dist-upgrade -y ca-certificates curl \
-    && apt install -y python3-setuptools libpython3.9 python3.9-dev \
+RUN apt-get install -y curl python3-setuptools libpython3.9 python3.9-dev \
     && curl https://bootstrap.pypa.io/get-pip.py | python3 \
     && pip3 install --only-binary ":all:" grpcio protobuf==3.20.1
 
+
 # Remove some things to decrease CVE surface
-RUN  apt remove -y --allow-remove-essential libtiff5 ncurses-base \
+RUN  apt-get remove -y --allow-remove-essential libtiff5 ncurses-base \
 	&& rm /usr/bin/passwd && rm /usr/sbin/adduser
 
 # Clean up caches, unwanted .a and .o files
 RUN rm -rf /root/.cache \
-    && apt -y autoremove \
-    && apt clean \
+    && apt-get -y autoremove \
+    && apt-get clean \
```

Comment:
In recent debian/ubuntu base images, `apt` is encouraged. As the from image didn't change into a legacy one, I'd prefer to see apt being used going forward. Not essential, if there's reason to leave it like this that's :+1: too

Reply:
That was mainly to align with the debian guidelines regarding `apt` usage in scripts.
https://manpages.debian.org/bullseye/apt/apt.8.en.html#SCRIPT_USAGE_AND_DIFFERENCES_FROM_OTHER_APT_TOOLS
There's a very slight chance that `apt` could break backward compatibility in future releases.